docker-compose.yml

```diff
@@ -202,7 +202,7 @@ services:
   #
   faf-website:
     container_name: faf-website
-    build: git://github.com/FAForever/website.git#v1.1.1.9
+    build: git://github.com/FAForever/website.git#v1.1.1.10
     image: faforever/faf-website
```

Comment:
The most important version tag is really the `image` key, since by default `docker-compose` will pull rather than build.

In general I'd kind of say we should leave `build` as tracking the default branch or `master`, and strictly update the tag of `image` instead-- then we don't have to duplicately update versions. @micheljung what're your thoughts on that?

Reply:
Imo that's the right approach, PR updated.

Reply:
I forgot, what exactly is the use case of `build` anyway?